### PR TITLE
missing translation for 'Add a new stack'

### PR DESCRIPTION
**Pull request by artemanufrij**
Signed-off-by: Artem Anufrij <artem.anufrij@live.de>

### CODE REVIEW

**Comment by artemanufrij** [comment]
@pixelipo you are right! Thanks!

File: templates/part.boardlist.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
 		<thead>
 		<tr>
 			<td class="cell-board-bullet"></td>
-			<td class="cell-board-title" width="90%"><?php p($l->t('Board title')); ?></td>
+			<td class="cell-board-title" width="90%"><?php p($l->t('Boards')); ?></td>
```
Comment (pixelipo) [inline templates/part.boardlist.php]: I don't think this is correct, @artemanufrij - it's a row title, and the row displays board titles.

Comment (artemanufrij) [inline templates/part.boardlist.php]: @juliushaertl what do you think about?

Comment (artemanufrij) [inline templates/part.boardlist.php]: @pixelipo @juliushaertl may be just "Title"?

Comment (pixelipo) [inline templates/part.boardlist.php]: `Title` is better.

By the way, Files app uses `Name` for the same row in their table

Comment (artemanufrij) [inline templates/part.boardlist.php]: @pixelipo ok, I change it to"Title"

Comment (juliushaertl) [inline templates/part.boardlist.php]: Yep just title sounds better.

File: templates/part.board.mainView.php [hunk 2]
```diff
@@ -34,10 +34,10 @@
 		<div class="stack" ng-repeat="s in stacks" data-as-sortable-item
 			 data-columnindex="{{$index}}" id="column{{$index}}"
 			 style="">
-			<h2 data-as-sortable-item-handle>
+			 <h2 data-as-sortable-item-handle>
```
Comment (pixelipo) [inline templates/part.board.mainView.php]: you got some weird space left before the `<h2>` tag

Comment (artemanufrij) [inline templates/part.board.mainView.php]: @pixelipo fixed

**Pull request merged**